src/sync-pending-events.ts

```diff
@@ -254,7 +255,15 @@ export const getweb3 = {
   }
 }
 
-async function getLatestBlockNumber(provider: string) {
+export async function getReliableBlockNumber(trueLatestBlockNumber: any, lastCheckedBlockNumber: number, bufferBlocks: number) {
```

Comment:
do we really need to export this (only for test?）

Reply:
the same problem exists for getValidBlockNumber.

Reply:
It is for unit test and looks like no better simple solution